integrations/terraform/envs/portal/config/environment_configuration/environments.yaml

```diff
@@ -0,0 +1,51 @@
+development:
+  AWS:
+    account_id: "012345678901"
+    region: eu-west-1
+    can_read_from:
+    - production
+    Glue:
```

Comment:
lowercase

Reply:
Fixed with latest commit. Platform and service names will always be exported as lowercase, also reflected this in all example files.